### PR TITLE
Remove module-unification blueprints

### PR DESCRIPTION
**Pull request by dcyriller**
Relates to https://github.com/ember-cli/ember-cli/issues/8790

I've made a search using ember-observer's code-search tool and the query `unification`. Here are the repos in which there are blueprints to remove:
https://github.com/emberjs/ember.js/
~~https://github.com/emberjs/data~~ EDIT: done
https://github.com/miguelcobain/ember-paper
https://github.com/mike-north/ember-qunit-decorators
~~https://github.com/kaliber5/ember-bootstrap~~ EDIT: done
https://github.com/ember-decorators/ember-decorators
https://github.com/rwjblue/sparkles-component

Non-related to the blueprints, but as it came out with the search, I'm writting it here. There is
- build related code in https://github.com/glimmerjs/glimmer-application-pipeline
- MU resolver https://github.com/ember-cli/ember-resolver
- blueprint test helpers https://github.com/ember-cli/ember-cli-blueprint-test-helpers

### CODE REVIEW

**Comment by dcyriller** [comment]
165 files changed 😬

**Comment by dcyriller** [comment]
> I wanted to confirm that this still allows applications to use MU (aka it doesn't do anything WRT the build support code), right?

My goal was to remove _only_ blueprints support. I've left the build code untouched. I've just confirmed that now manually.

To remove build support, the following files would be impacted:
- `lib/models/addon.js`
- `lib/models/project.js`
- `lib/broccoli/ember-app.js`
- `lib/broccoli/default-packager.js`

**Comment by dcyriller** [comment]
Changing the target from master to beta, I had to:
```sh
git reset --hard origin/beta
git cherry-pick the 6 commits hashes
```

**Comment by dcyriller** [comment]
Ci failure seems unrelated

**Comment by rwjblue** [comment]
> > Assuming that is correct (certainly seems correct to me from the diff), then we should land this just after branching master to beta (this will be in ~ a day or two).
>
> Changing the target from master to beta

😩 I'm really sorry for being unclear @dcyriller. I was meaning that this PR would still target master, but would wait to be merged until master represented 3.15 which won't happen until 3.13 is released as stable, and 3.14 (current master) has made its way into beta. 

tldr; can you target this again at `master` (again, I'm really sorry), and we will merge in ~ a week or so once 3.13.0 is released.

**Comment by dcyriller** [comment]
ahah ok, no worries @rwjblue my english can be rough sometimes

**Comment by rwjblue** [comment]
Looks like this test is failing when the MU feature flag is enabled:

> not ok 1335 Acceptance: addon-smoke-test works in most common scenarios for an example addon

**Comment by rwjblue** [comment]
OK, _finally_ ready to land this into master. I've just rebased again 🤞.

**Comment by rwjblue** [comment]
😭
